src/components/organisms/EventModal/EventModal.tsx

```diff
@@ -94,6 +94,7 @@ export const EventModal: React.FC<EventModalProps> = ({
 
   return (
     <Modal show={show} onHide={onHide} className="EventModal">
+      <Modal.Header className="EventModal__close" closeButton />
```

Comment:
Should this be added to all of the `Modal`s? I only see it on 1, I would expect to see a LOT of this to fix it across the platform.